From b4ed78c642bedc03cb392e5ef20c76dbb2c0c75c Mon Sep 17 00:00:00 2001
Message-Id: <b4ed78c642bedc03cb392e5ef20c76dbb2c0c75c.1417780878.git.chang-joon.lee@intel.com>
In-Reply-To: <cb098d33cdac3271103834fbce08218248d7bddb.1417780878.git.chang-joon.lee@intel.com>
References: <cb098d33cdac3271103834fbce08218248d7bddb.1417780878.git.chang-joon.lee@intel.com>
From: Durgadoss R <durgadoss.r@intel.com>
Date: Wed, 19 Nov 2014 17:00:18 +0530
Subject: [PATCH 19/34] REVERTME [VPG]: drm/i915: Enable PSR by default

This patch changes the default for i915.enable_psr
to 1, enabling psr for all targets unless forced off
on the command line.

This change affects CHV only. The code prevents VLV from
enabling psr.

REVERTME: The correct place to do this is in the kernel
          boot params, but this will break BXT. We should
          revert this patch and set the param globally as
          soon as it's safe to do so on all targets.

Issue: GMINL-3876
Change-Id: I8b548dfa7c7385d6fa33335ec71aae71849e4448
Signed-off-by: Durgadoss R <durgadoss.r@intel.com>
---
 drivers/gpu/drm/i915/i915_params.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_params.c b/drivers/gpu/drm/i915/i915_params.c
index 7c3a76d..c92e804 100644
--- a/drivers/gpu/drm/i915/i915_params.c
+++ b/drivers/gpu/drm/i915/i915_params.c
@@ -39,7 +39,7 @@ struct i915_params i915 __read_mostly = {
 	.enable_execlists = -1,
 	.enable_hangcheck = true,
 	.enable_ppgtt = -1,
-	.enable_psr = 0,
+	.enable_psr = 1,
 	.preliminary_hw_support = IS_ENABLED(CONFIG_DRM_I915_PRELIMINARY_HW_SUPPORT),
 	.disable_power_well = 1,
 	.enable_ips = 1,
@@ -134,7 +134,7 @@ MODULE_PARM_DESC(enable_execlists,
 	"(-1=auto [default], 0=disabled, 1=enabled)");
 
 module_param_named(enable_psr, i915.enable_psr, int, 0600);
-MODULE_PARM_DESC(enable_psr, "Enable PSR (default: false)");
+MODULE_PARM_DESC(enable_psr, "Enable PSR (default: true)");
 
 module_param_named(preliminary_hw_support, i915.preliminary_hw_support, int, 0600);
 MODULE_PARM_DESC(preliminary_hw_support,
-- 
1.7.9.5

